### PR TITLE
#6Task. peter.shumskiy

### PR DESCRIPTION
**Pull request by petr-shumskiy**
https://travis-ci.org/github/petr-shumskiy/js-assignments/builds/709546493

### CODE REVIEW

File: task/06-conditions-n-loops-tasks.js [hunk 1]
```diff
@@ -30,7 +30,9 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    return num % 3 === 0 
+    ? num % 5 === 0 ? 'FizzBuzz' : 'Fizz'
+    : num % 5 === 0 ? 'Buzz' : num
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: Please, avoid this code style, if-else is more readable than multiple conditional operators ? :

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

File: task/06-conditions-n-loops-tasks.js [hunk 2]
```diff
@@ -150,7 +164,7 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+    return (point.x - circle.center.x)**2 + (point.y - circle.center.y)**2 < circle.radius**2
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: Please, spaces around operators

**Comment by petr-shumskiy** [comment]
[link](https://travis-ci.org/github/petr-shumskiy/js-assignments/jobs/722873097) to new build

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**